lib/broccoli/ember-app.js

```diff
@@ -827,16 +827,36 @@ EmberApp.prototype.appAndDependencies = function() {
 
   this._addAppTests(sourceTrees);
 
-  var emberCLITree = this._processedEmberCLITree();
+  var es6app = this._es6ifyAppModules(mergeTrees(sourceTrees, {
+    overwrite: true,
+    description: 'TreeMerger (appModules)'
+  }));
 
-  sourceTrees.push(emberCLITree);
+  var emberCLITree = this._processedEmberCLITree();
```

Comment:
Likewise, I'm assuming that the processedEmberCLITree doesn't need to go through `_es6ifyAppModules`.